src/content/docs/apis/nerdgraph/get-started/nerdgraph-explorer.mdx

```diff
@@ -8,56 +8,39 @@ metaDescription: How to use the New Relic NerdGraph API explorer.
 redirects:
 ---
 
-import apisanddataNerdGraphCreateUserKey from 'images/apis-and-data_screenshot-crop_NerdGraph-create-user-key.webp'
 
-import apisanddataNerdGraphInternalDocs from 'images/apis-and-data_screenshot-crop_NerdGraph-internal-docs.webp'
+import apisScreenshotAPIKeyInNerdGraph from 'images/apis_screenshot-crop_Screenshot-API-key-in-NerdGraph.webp'
 
-import apisanddataNerdGraphMutationExample from 'images/apis-and-data_screenshot-full_NerdGraph-mutation-example.webp'
+import apisNerdGraphInternalDocs from 'images/apis-and-data_screenshot-crop_NerdGraph-internal-docs.webp'
 
-import apisanddataNerdGraphToolsMenu from 'images/apis-and-data_screenshot-crop_NerdGraph-tools-menu.webp'
+import apisNerdGraphMutationExample from 'images/apis-and-data_screenshot-full_NerdGraph-mutation-example.webp'
 
+import apisNerdGraphToolsMenu from 'images/apis-and-data_screenshot-crop_NerdGraph-tools-menu.webp'
 
-NerdGraph is our main API for querying data and making configurations to New Relic features. One popular way to use NerdGraph and make API requests is with our NerdGraph explorer. 
 
-## Find the explorer [#find]
 
-To find the NerdGraph explorer: 
+NerdGraph is our GraphQL API you can use to query data and make configuration changes in New Relic features. You can send GraphQL to the NerdGraph API in a variety of ways: from your code, with curl or CLI requests, and in the NerdGraph API explorer. The explorer is a GUI GraphQL tool (based on GraphiQL) where you can experiment with queries and mutations (changes) in NerdGraph.
 
-* EU: If your New Relic organization uses an [EU data center](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers), go to [api.eu.newrelic.com/graphiql](https://api.eu.newrelic.com/graphiql).
-* Outside EU: If your organization doesn't use an EU data center, go to [api.newrelic.com/graphiql](https://api.newrelic.com/graphiql).
+If you're new to GraphQL, get acquainted with the GraphQL API via the NerdGraph API explorer. If you need an overview of the API before starting this tutorial, check out [Meet NerdGraph: our GraphQL API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph).
 
-## What is the NerdGraph explorer? [#overview]
+## Step 1. Set up the NerdGraph API explorer [#set-up]
 
-Our NerdGraph API is built using GraphQL, a third-party query language. Our NerdGraph explorer is built using GraphiQL, which is an open source graphical interface for forming and making GraphQL requests. 
+1. Go to **[one.newrelic.com](https://one.newrelic.com)**.
+2. In the left pane, click **Apps**.
```

Comment:
we should send them to 'All capabilities' page because 'Apps' won't necessarily be on their left pane unless they've added it.

Reply:
@zuluecho9 good catch!

Reply:
@zuluecho9 OK, I changed this.